stream-video-android-core/src/main/kotlin/io/getstream/video/android/core/CallState.kt

```diff
@@ -306,6 +316,22 @@ public class CallState(
     private val _settings: MutableStateFlow<CallSettingsResponse?> = MutableStateFlow(null)
     public val settings: StateFlow<CallSettingsResponse?> = _settings
 
+    private val _durationInMs = flow {
+        while (scope.isActive) {
+            delay(1000)
+            val started = _session.value?.startedAt
+            val ended = _session.value?.endedAt ?: OffsetDateTime.now()
+            val difference = if (started == null) null else {
+                ended.toInstant().toEpochMilli() - started.toInstant()?.toEpochMilli()!!
+            }
+            emit(difference)
+        }
+    }
+
+    /** how many MS the call has been running, null if the call didn't start yet */
```

Comment:
Instead of `MS` I would either use `ms` or maybe it will be more clear with `milliseconds`